Governance/tech/conventions.md

```diff
@@ -1,147 +1,295 @@
-# Applying conventions to your files
+useful summary: https://cfconventions.org/Data/cf-documents/requirements-recommendations/conformance-1.9.html
+# Writing valid and compliants netCDF files
```

Comment:
*compliant

Reply:
thanks, I also forgot to remove the useful link, before the summary, that was there for my own sake, I now moved it at the end